src/App.tsx

```diff
@@ -12,20 +12,20 @@ import EditEvent from "./views/EditEvent";
 function App() {
   const location = useLocation();
   const [displayLocation, setDisplayLocation] = useState(location);
-  const [transitionStage, setTransistionStage] = useState("fadeIn");
+  const [transitionStage, setTransitionStage] = useState("fadeIn");
   useEffect(() => {
-    if (location !== displayLocation) setTransistionStage("fadeOut");
+    if (location !== displayLocation) setTransitionStage("fadeOut");
   }, [location, displayLocation]);
 
   return (
     <Box sx={{ position: "relative" }}>
       <NavBar />
       <Container
         sx={{ mt: 0.2, mb: 3, display: "flex", flexDirection: "column" }}
-        className={`${transitionStage}`}
+        className={transitionStage}
```

Comment:
no reason to use an interpolated string with a single value of type string, instead of just returning the value itself.